x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/search_ui.tsx

```diff
@@ -9,16 +9,50 @@ import React, { useEffect } from 'react';
 
 import { useActions } from 'kea';
 
-import { EuiPageHeader, EuiPageContentBody } from '@elastic/eui';
+import {
+  EuiPageHeader,
+  EuiPageContentBody,
+  EuiText,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiSpacer,
+  EuiLink,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
 
 import { FlashMessages } from '../../../shared/flash_messages';
 import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
 
+import { DOCS_PREFIX } from '../../routes';
 import { getEngineBreadcrumbs } from '../engine';
 
+import { SearchUIForm } from './components/search_ui_form';
+import { SearchUIGraphic } from './components/search_ui_graphic';
 import { SEARCH_UI_TITLE } from './constants';
 import { SearchUILogic } from './search_ui_logic';
 
+const DESCRIPTION = i18n.translate('xpack.enterpriseSearch.appSearch.engine.searchUI.description', {
+  defaultMessage: 'Preview search or kickstart your next search experience.',
+});
+
+const SEARCH_UI_BODY_TEXT = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.searchUI.bodyText',
```

Comment:
[optional] I could be wrong but I _think_ this should still use the `Description` suffix since it's in a `<p>` tag (ref: https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types)
```suggestion
  'xpack.enterpriseSearch.appSearch.engine.searchUI.bodyDescription',
```